tests/TestFuture.re

```diff
@@ -154,9 +153,10 @@ describe("Future Belt.Result", () => {
   });
 
   test("flatMapOk", () => {
-    Belt.Result.Ok("four")
-    |. Future.value
-    |. Future.flatMapOk(s => Belt.Result.Ok(s ++ "!") |. Future.value)
+    (
+      Belt.Result.Ok("four") |. Future.value
+      >>= (s => Belt.Result.Ok(s ++ "!") |. Future.value)
```

Comment:
I'm not sure this add clarity to the code here. (Also, same admonishment as above.)  Perhaps there's a more clear example which shows the ergonomic gains of the infix operators.